extensions/amp-analytics/0.1/visibility-impl.js

```diff
@@ -16,23 +16,34 @@
 
 import {dev} from '../../../src/log';
 import {getService} from '../../../src/service';
+import {rectIntersection} from '../../../src/layout-rect';
 import {resourcesFor} from '../../../src/resources';
 import {timer} from '../../../src/timer';
 import {user} from '../../../src/log';
 import {viewportFor} from '../../../src/viewport';
+import {viewerFor} from '../../../src/viewer';
+import {VisibilityState} from '../../../src/service/viewer-impl';
```

Comment:
We should either move it into `viewer.js` or into its own `visibility-state.js`.